src/services/TransactionUpdater.tsx

```diff
@@ -1,6 +1,4 @@
-import React, { useEffect, useMemo } from 'react'
-import { toast } from 'react-toastify'
```

Comment:
Lets leave this library in for now, since we might use it for bridged asset notifications